packages/runtime/src/components/builtin/Text/EditableText.tsx

```diff
@@ -53,6 +54,15 @@ const EditableText = forwardRef(function EditableText(
     useState<DescriptorsPropControllers<Descriptors> | null>(null)
   const controller = propControllers?.text
 
+  useEffect(() => {
+    const element = editor?.findDOMNode([])
+    if (element == null || controller == null) return
+    return pollBoxModel({
+      element: element as unknown as HTMLElement,
```

Comment:
I would suggest checking the element is actually an HTML element instead of bypassing the types